astro.config.mjs

```diff
@@ -1,70 +1,85 @@
-import { defineConfig } from 'astro/config';
-import yaml from '@rollup/plugin-yaml';
-import icon from "astro-icon";
-import Compress from "astro-compress";
-import tailwind from "@astrojs/tailwind";
-import { remarkReadingTime } from "./src/plugins/remark-reading-time.mjs";
-import rehypeKatex from "rehype-katex";
-import Color from 'colorjs.io';
-import remarkMath from "remark-math";
-import rehypeAutolinkHeadings from "rehype-autolink-headings";
-import rehypeSlug from "rehype-slug";
+import tailwind from '@astrojs/tailwind'
+import yaml from '@rollup/plugin-yaml'
+import Compress from 'astro-compress'
+import icon from 'astro-icon'
+import { defineConfig } from 'astro/config'
+import Color from 'colorjs.io'
+import rehypeAutolinkHeadings from 'rehype-autolink-headings'
+import rehypeKatex from 'rehype-katex'
+import rehypeSlug from 'rehype-slug'
+import remarkMath from 'remark-math'
+import { remarkReadingTime } from './src/plugins/remark-reading-time.mjs'
 
-const oklchToHex = function (str) {
-  const DEFAULT_HUE = 250;
-  const regex = /-?\d+(\.\d+)?/g;
-  const matches = str.string.match(regex);
-  const lch = [matches[0], matches[1], DEFAULT_HUE];
-  return new Color("oklch", lch).to("srgb").toString({
-    format: "hex"
-  });
-};
+const oklchToHex = str => {
+  const DEFAULT_HUE = 250
+  const regex = /-?\d+(\.\d+)?/g
+  const matches = str.string.match(regex)
+  const lch = [matches[0], matches[1], DEFAULT_HUE]
+  return new Color('oklch', lch).to('srgb').toString({
+    format: 'hex',
+  })
+}
 
 // https://astro.build/config
 export default defineConfig({
   site: 'https://fuwari.vercel.app/',
   base: '/',
-  integrations: [tailwind(), icon({
-    include: {
-      'material-symbols': ['*'],
-      'fa6-brands': ['*'],
-      'fa6-regular': ['*'],
-      'fa6-solid': ['*']
-    }
-  })],
+  integrations: [
+    tailwind(),
+    icon({
+      include: {
+        'material-symbols': ['*'],
+        'fa6-brands': ['*'],
+        'fa6-regular': ['*'],
+        'fa6-solid': ['*'],
+      },
+    }),
+    Compress({
+      Image: false,
```

Comment:
In the first place, it is not used for image compression, and is only introduced to minify js.